spec/requests/api_requests_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+describe "get index request " do 
```

Comment:
Consider adding tests for:

- invalid apikey prevents user from deleting a todo
- an attacker with a **valid** apikey cannot delete a victim's todo